src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -19,209 +19,470 @@ If you're new to the topic of running a node, or looking for a less technical pa
+
+#### Getting the client software {#getting-the-client}
+
+First, you need to obtain your preferred [execution client](/developers/docs/nodes-and-clients/#execution-clients) and [consensus client](developers/docs/nodes-and-clients/#consensus-clients) software. 
+
+You can simply download an executable application or installation package that suits your operating system and architecture. Always verify the signatures and checksums of downloaded packages. Some clients also offer repositories or Docker images for easier installation and updates.All of the clients are open source, so you can also build them from source. This is a more advanced method, but in some cases, it might be required. 
+
+Instructions for installing each client are provided in the documentation linked in the client lists above. 
+
+Here are the release pages of clients where you can find their pre-built binaries or instructions on installation: 
+
+##### Execution clients
 
 - [Geth](https://geth.ethereum.org/downloads/)
-- [OpenEthereum,](https://github.com/openethereum/openethereum/releases)
 - [Nethermind](https://downloads.nethermind.io/)
-- [Besu](https://pegasys.tech/solutions/hyperledger-besu/)
-- [Erigon](https://github.com/ledgerwatch/erigon)
+- [Besu](https://github.com/hyperledger/besu/releases)
+- [Erigon](https://github.com/ledgerwatch/erigon#usage) (Doesn't provide a pre-built binary, has to be compiled)
 
 It is also worth noting that client diversity is an [issue on the execution layer](https://clientdiversity.org/),
-with Geth being run on a supermajority (>66%) of all Ethereum nodes. It is recommended that readers on this page consider
-running a minority execution client.
+with Geth being run on a supermajority (>66%) of all Ethereum nodes. It is recommended that readers on this page consider running a minority execution client.
+
+##### Consensus clients
+
+- [Lighthouse](https://github.com/sigp/lighthouse/releases/latest)
+- [Teku](https://github.com/ConsenSys/teku/releases)
+- [Nimbus](https://github.com/status-im/nimbus-eth2/releases/latest)
+- [Prysm](https://github.com/prysmaticlabs/prysm/releases/latest)
+- [Lodestar](https://chainsafe.github.io/lodestar/install/source/) (Doesn't provide a pre-built binary, only a Docker image or to be build from source)
+
+There is currently a [client diversity](/developers/docs/nodes-and-clients/client-diversity/) issue where a large dominance of Prysm clients poses a risk to the health of the network. In response to the initial drive to even out the client diversity many Prysm nodes switched to Lighthouse to the extent that it now also has a problematic market share. It is therefore recommended to consider choosing a minority client. 
```

Comment:
I would avoid mentioning specific client too much for this info to not get out of date. Can point people towards the data and say "use a minority client" :-)

Reply:
This as added by eth.org team and I wanted to keep it as it is still up to date but I agree with the change to minimize needed updates